server/routes/updateGoal/mappers/goalToUpdateGoalFormMapper.ts

```diff
@@ -22,12 +25,12 @@ const toUpdateStepForm = (step: Step): UpdateStepForm => {
     reference: step.stepReference,
     title: step.title,
     stepNumber: step.sequenceNumber,
-    status: step.status,
+    status: StepStatusValue[step.status as keyof typeof StepStatusValue],
   }
 }
 
 const toDateString = (date: Date): string => {
-  return moment(date).format('YYYY-MM-DD')
+  return format(date, 'yyyy-MM-dd')
```

Comment:
As well as the mappers mapping the status field, they also map dates. 
Previously we used the `moment` date library, but we have a ticket on the backlog to migrate from `moment` to `date-fns`. Given that I was making a change to these mappers anyway I took the opportunity to use `date-fns` date functions here 👍